### DISCUSSION TITLE
(DEGEN CHAİN) coin I swapped is not in my wallet

### DISCUSSION THREAD

**Discussion by fozruk0** [blockscout/blockscout#9767] [Support]
hello,

via https://dex.swapdegen.tips/#/swap

I exchanged my degen coins with SAITAMA, txn is below;

https://explorer.degen.tips/tx/0x538b5cf45a80a13efd5652f1a187b8ae7961340c59d9808eed7aec5fa4548f46

This is my 5th from last transaction. I should have 99 SAITAMAs in my wallet but I don't.

Afterwards, there is no action to remove my SAITAMAs from the wallet, so why are there no SAITAMAs in my wallet? can you help me

![image](https://github.com/blockscout/blockscout/assets/165609759/29c29d37-84ce-4ac3-8101-3c1b6bfd0f1b)

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
